### PR TITLE
updating documentation

### PR DESCRIPTION
**Pull request by rmolin88**
I think it will also be helpful if a some information was added to the wiki or readme on how get autocomplete for android classes and created classes for and android app created under android-studio.

Anyways, awesome plugin!

### CODE REVIEW

**Comment by DonnieWest** [comment]
Are you having problems getting it working? It should "just work" when you open an Android project that uses the Gradle build system as long as you're in the root of the project.

But yes, documentation would be nice. Would you mind contributing some based on your experience?

**Comment by rmolin88** [comment]
I wouldnt mind at all contributing to documentation. The thing is that I am very new to the java/android world. But yeah right now for me doesnt work. Specially VimStudio where I can no even call the command line commands like SetupVimStudio. I am not using Gradle to build I use Android-Studio. I guess I just need to learn more but I wouldnt mind documenting my learning process if I have some guidance on how to get it to work.

**Comment by DonnieWest** [comment]
Let's see if we can get you going then:

Assuming a fresh project from Android Studio, In your terminal:
1. cd into the root of your project
2. Look for a file called build.gradle to make sure you're using the new Gradle build system (Android Studio should have automatically created it for you and does this by default)
3. run `./gradlew assembleDebug` to make sure the project builds properly. If it doesn't, that needs to be fixed before you can proceed.
4. Launch Neovim
5. Open a java file
6. Check if completions work

Please let me know how that goes for you :)

As far as VimStudio goes:

Sorry about the terrible documentation (and experience) that VimStudio provides. I'm hoping to get it cleaned up soon! If you've followed the steps above for troubleshooting vim-javacomplete2 and those work but VimStudio commands don't, please run `UpdateRemotePlugins` from within Neovim and restart Neovim. The commands should now be present, but, if you're still having problems then please open an issue over there and I'll see how I can help :)

**Comment by rmolin88** [comment]
## Steps Above
I've tried the steps but still doesnt work. [here](https://github.com/rmolin88/vimrc) 
is my vimrc with name **\_vimrc**  and [here](https://github.com/rmolin88/BaseballScoreApp.git) 
is a link to my app.
I am able to autocomplete functions `R.*`, `LinearLayout` but not my own created functions.
## Things I've tried:
- Created and loaded ctags on vim $ANDROID\_SDK/sources
- Created tags on my app source code
- Pointed CLASSPATH to android-24/android.jar. This got me a little of
	autocomplete working, like LinearLayout and stuff.
- `
let g:JavaComplete_SourcesPath='~/Documents/seafile-client/Seafile/KnowledgeIsPower/udacity/android-projects/BaseballScore/app/build/generated/source/r/'`
## VimStudio
- Thanks for the note on `UpdateRemotePlugins` now I got the commands to work.
## Neomake 
- Doesnt work either
- MakeJob function fails with message `"gradle" is not an executable`

## Important:
I would to thank all of you guys of taking the time of making these great plugins that are super 
useful, and then on top of that taking to time to help somebody get it working.

**Comment by DonnieWest** [comment]
Looking over your vimrc, your hard coded sources path [here](https://github.com/rmolin88/vimrc/blob/master/_vimrc#L1571) _might_ be interfering. Honestly though, it would probably be easiest to work this out if we used a minimal configuration. Here's one I just made:

```
" Install Vim Plug if not installed
if empty(glob('~/.config/nvim/autoload/plug.vim'))
  silent !curl -fLo ~/.config/nvim/autoload/plug.vim --create-dirs
    \ https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
  autocmd VimEnter * PlugInstall
endif

call plug#begin()

Plug 'artur-shaik/vim-javacomplete2', { 'branch': 'master' }

call plug#end()

autocmd FileType java setlocal omnifunc=javacomplete#Complete
```
I went ahead and cloned your project down on my local machine. It seems to work when using my normal vimrc which you can find [here](https://github.com/DonnieWest/dotfiles/blob/dotfiles/vimrc) and the minimal config found above. Here's also a rather bad [screencast of it in action](https://asciinema.org/a/2p71rpp74z3hpehgj21jg9v16) just in case that helps somehow. 

Also, my pleasure! It's nice to have guys around like @artur-shaik who gives us the superpowers :+1: not to mention the many contributions from others

**Comment by rmolin88** [comment]
Hey I got it to Work!!! :satisfied:. The trikcy part is that neovim must be
launched at the project root. The reason why it wasnt working for me is that I
normally use neovim-qt and then just navigate to the code. If you this doesnt
work. But if I go command line straight to the project root like you showed on
the screen cast and open there it
works. I dont know if this is design but its kinda a trikcy thing.
Anyways thank you very much for the help. Please let me know if you guys want me
to document this somewhere.
BTW: vim-plug rocks :metal:

File: doc/javacomplete.txt [hunk 1]
```diff
@@ -315,6 +315,11 @@ A single letter indicates the kind of compeltion item. These kinds are:
         let g:JavaComplete_UsePython3 = 1
         an option to make javacomplete2 use python3 interpreter instead of python2, default value is 0.
 
+	Disable javacomplete2 default mappings:
```
Comment (artur-shaik) [inline doc/javacomplete.txt]: Seems wrong indentation.
Can you, please, ident lines the same as previous options in doc:

1. let NameOfParameter
2. descsription text.

**Pull request merged**